L1Trigger/L1THGCal/test/HGCalBackendStage1ParameterExtractor.cc

```diff
@@ -40,7 +40,7 @@ class HGCalBackendStage1ParameterExtractor : public edm::stream::EDAnalyzer<> {
   uint32_t getPhiBin(uint32_t roverzbin, double phi);
   double rotatedphi(double phi, int sector);
 
-  uint32_t getReducedModuleHash(int subdet, int layer, int ueta, int vphi);
+  uint32_t getReducedModuleHash(HGCalTriggerModuleDetId moduleId);
```

Comment:
Better to pass by const reference rather than value to avoid the copy

Reply:
done